### PR TITLE
Add bibtex entries for research publications

### PR DESCRIPTION
**Pull request by Animaholic**
## Checks

- [ ] I've used [pre-commit](https://microsoft.github.io/FLAML/docs/Contribute#pre-commit) to lint the changes in this PR, or I've made sure [lint with flake8](https://github.com/microsoft/FLAML/blob/816a82a1155b4de4705b21a615ccdff67c6da379/.github/workflows/python-package.yml#L54-L59) output is two 0s.
- [ ] I've included any doc changes needed for https://microsoft.github.io/FLAML/. See https://microsoft.github.io/FLAML/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/Research.md [hunk 1]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
 ```
 
 * [Frugal Optimization for Cost-related Hyperparameters](https://arxiv.org/abs/2005.01571). Qingyun Wu, Chi Wang, Silu Huang. AAAI 2021.
+
+```bibtex
+@inproceedings{wang2021flaml,
```
Comment (sonichi) [inline website/docs/Research.md:19]:
```suggestion
@inproceedings{wu2021cfo,
```

File: website/docs/Research.md [hunk 2]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
...
+    author={Qingyun Wu and Chi Wang and Silu Huang},
+    year={2021},
+    booktitle={AAAI},
+}
+```
+
 * [Economical Hyperparameter Optimization With Blended Search Strategy](https://www.microsoft.com/en-us/research/publication/economical-hyperparameter-optimization-with-blended-search-strategy/). Chi Wang, Qingyun Wu, Silu Huang, Amin Saied. ICLR 2021.
+
+```bibtex
+@inproceedings{wang2021flaml,
```
Comment (sonichi) [inline website/docs/Research.md:30]:
```suggestion
@inproceedings{wang2021blendsearch,
```

File: website/docs/Research.md [hunk 3]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
...
+    author={Chi Wang and Qingyun Wu and Silu Huang and Amin Saied},
+    year={2021},
+    booktitle={ICLR},
+}
+```
+
 * [An Empirical Study on Hyperparameter Optimization for Fine-Tuning Pre-trained Language Models](https://aclanthology.org/2021.acl-long.178.pdf). Susan Xueqing Liu, Chi Wang. ACL 2021.
+
+```bibtex
+@inproceedings{wang2021flaml,
```
Comment (sonichi) [inline website/docs/Research.md:41]:
```suggestion
@inproceedings{liuwang2021hpolm,
```

File: website/docs/Research.md [hunk 4]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
...
+    author={Susan Xueqing Liu and Chi Wang},
+    year={2021},
+    booktitle={ACL},
+}
+```
+
 * [ChaCha for Online AutoML](https://www.microsoft.com/en-us/research/publication/chacha-for-online-automl/). Qingyun Wu, Chi Wang, John Langford, Paul Mineiro and Marco Rossi. ICML 2021.
+
+```bibtex
+@inproceedings{wang2021flaml,
```
Comment (sonichi) [inline website/docs/Research.md:52]:
```suggestion
@inproceedings{wu2021chacha,
```

File: website/docs/Research.md [hunk 5]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
...
+    author={Qingyun Wu and Chi Wang and John Langford and Paul Mineiro and Marco Rossi},
+    year={2021},
+    booktitle={ICML},
+}
+```
+
 * [Fair AutoML](https://arxiv.org/abs/2111.06495). Qingyun Wu, Chi Wang. ArXiv preprint arXiv:2111.06495 (2021).
+
+```bibtex
+@inproceedings{wang2021flaml,
```
Comment (sonichi) [inline website/docs/Research.md:63]:
```suggestion
@inproceedings{wuwang2021fairautoml,
```

**Comment by Animaholic** [comment]
> @microsoft-github-policy-service agree [company="{N/A}"]

File: website/docs/Research.md [hunk 6]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
...
+    author={Qingyun Wu and Chi Wang},
+    year={2021},
+    booktitle={ArXiv preprint arXiv:2111.06495},
+}
+```
+
 * [Mining Robust Default Configurations for Resource-constrained AutoML](https://arxiv.org/abs/2202.09927). Moe Kayali, Chi Wang. ArXiv preprint arXiv:2202.09927 (2022).
+
+```bibtex
+@inproceedings{wang2021flaml,
```
Comment (sonichi) [inline website/docs/Research.md:74]:
```suggestion
@inproceedings{kayaliwang2022default,
```

**Review by sonichi — COMMENTED** [review]
Thanks. I'll approve after the requested changes.

File: website/docs/Research.md [hunk 7]
```diff
@@ -14,8 +14,67 @@ For technical details, please check our research publications.
...
+    title={Mining Robust Default Configurations for Resource-constrained AutoML},
+    author={Moe Kayali and Chi Wang},
+    year={2021},
```
Comment (sonichi) [inline website/docs/Research.md:77]:
```suggestion
    year={2022},
```

**Review by qingyun-wu — APPROVED** [review]

**Review by skzhang1 — APPROVED** [review]

**Review by ZviBaratz — APPROVED** [review]

**Comment by skzhang1** [comment]
@Animaholic Can you sign the agreement? Thanks

**Review by mkyl — APPROVED** [review]

**Comment by Animaholic** [comment]
@microsoft-github-policy-service agree

**Pull request merged**